tock-process-console/src/ui_management/pages/terminal_page/components/terminal_box.rs

```diff
@@ -13,7 +13,8 @@ use crossterm::event::{KeyCode, KeyEventKind, MouseEventKind};
 use ratatui::{
     layout::{Alignment, Rect},
     style::{Color, Modifier, Style},
-    widgets::{Block, Borders, Paragraph},
+    widgets::block::Block,
+    widgets::{Borders, Paragraph},
```

Comment:
What does this modification serve?

Reply:
The reason I made this modification is because it was suggested when running "cargo build --verbose": 
error[E0308]: mismatched types
   --> tock-process-console/src/ui_management/pages/terminal_page/components/terminal_box.rs:255:82
    |
255 |                 let pseudo_terminal = PseudoTerminal::new(parser.screen()).block(block);
    |                                                                            ----- ^^^^^ expected `ratatui::widgets::block::Block<'_>`, found `ratatui::widgets::Block<'_>`